device/protocol_mqtt_v1.go

```diff
@@ -100,6 +100,9 @@ func (d *Device) initializeMQTTClient(brokerAddress string) error {
 			}
 
 			if iface, ok := d.interfaces[interfaceName]; ok {
+				// Ensure all defaults for the interface
+				iface = interfaces.EnsureInterfaceDefaults(iface)
```

Comment:
This should be done once when the interface is added instead of here

Reply:
Removing it from here.